css-scroll-anchoring-1/Overview.bs

```diff
@@ -97,7 +97,7 @@ following criteria:
 
 Some elements are considered to be <dfn
 id="anchor-priority-candidates">priority candidates</dfn> for anchor selection:
-  1. An element that <a href="https://html.spec.whatwg.org/multipage/semantics-other.html#selector-focus">has focus</a>.
+  1. An element that <a href="https://html.spec.whatwg.org/multipage/interaction.html#dom-document-activeelement">has focus</a>.
```

Comment:
Probably also not what you want to link to, for two reasons:

 * It retargets shadow DOM nodes, so you avoid anchoring focused nodes inside shadow trees and anchoring to the host instead.

 * It returns the `<body>` element, which means that if you have no focus, you only anchor to the body, which is definitely not what you want.

Reply:
I've spent some time looking for the focused element I mean, and I'm starting to realize that focus is difficult.

I think this might be the another definition here:
https://www.w3.org/TR/html51/editing.html#focused

It does seem to talk about retargetting to the document, but it seems that it's meant to ensure that the focused element is the deepest element that matches the given criteria. It does seem to default to the document if there is no selection.

I think the path forward here is to point to one of these focus definitions but also say that we should exclude the body/document from being a potential priority candidate.

Or maybe just explicitly say that it is a deepest element that has focus (ie responds to the :focus selector)

What do you think?

Reply:
We shouldn't link to the w3c html spec. The whatwg version is the right one :)

We should probably just say that this is "The DOM anchor of the focused area of the document.", where "focused area" is https://html.spec.whatwg.org/multipage/interaction.html#focused-area-of-the-document and "DOM anchor" is https://html.spec.whatwg.org/multipage/interaction.html#dom-anchor.

That's effectively the unretargeted active element, without falling back to `<body>`. Now that's quite a mouthful. Maybe @annevk / @domenic can export that in a way that it has a nicer name from html?

Reply:
Thanks for helping me out with the text!

I got hung up on "focused element" as opposed to "focused area". The text you have makes sense, and I've updated to that.